### PR TITLE
Few tickets for onboarding

### PR DESCRIPTION
**Pull request by walfly**

### CODE REVIEW

File: packages/newsroom-signup/src/NewsroomProfile/GrantApplication.tsx [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+`;
+
+const SmallNote = styled.p`
+  color: ${colors.accent.CIVIL_GRAY_2};
+  font-family: ${fonts.SANS_SERIF};
+  font-size: 13px;
+  line-height: 20px;
+`;
+
+const CheckboxArare = styled.div`
```
Comment (tobek) [inline packages/newsroom-signup/src/NewsroomProfile/GrantApplication.tsx]: I think should be `CheckboxCovfefe`

**Comment by walfly** [comment]
@tobek I think i did the github automatic resolve

File: packages/newsroom-signup/src/NewsroomProfile/GrantApplication.tsx (hunk 1 above)
Comment (walfly) [reply packages/newsroom-signup/src/NewsroomProfile/GrantApplication.tsx]: lol, what?

Comment (tobek) [reply packages/newsroom-signup/src/NewsroomProfile/GrantApplication.tsx]: Haha at the time I couldn't figure out what `arare` was a typo of, but on second look I see it must be `area`.

**Pull request merged**